docs/tutorials/configure-external-signer-tls.md

```diff
@@ -24,7 +24,7 @@ This tutorial configures TLS between Teku and Web3Signer, and use the [`keytool`
 - [Java `keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html).
 - A running execution client such as [Hyperledger Besu], or cloud-based service such as [Infura] synced to the Goerli testnet.
 
-This tutorial connects to an ETH2 testnet, and uses [Infura] to access the Goerli ETH1 testnet. You can sign up for a free [Infura] account, or you can start your own [ETH1 Goerli node].
+This tutorial connects to an Ethereum testnet for the consensus layer and uses Infura to access the Goerli testnet for the execution layer. You can sign up for a free [Infura] account, or you can set up your own [Goerli node] node for the execution layer, which uses [Infura] to access the Goerli testnet.
```

Comment:
I havne't played with this in ages, but I'm not sure infura offers this access any more...